src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Cache/CacheContextEntry.cs

```diff
@@ -8,7 +8,8 @@ public CacheContextEntry(string key, string value)
             Value = value;
         }
 
-        public string Key { get; set; }
-        public string Value { get; set; }
+        public string Key { get; }
+
+        public string Value { get; }
     }
```

Comment:
The readonly could be applied to the whole struct.

    public readonly struct CacheContextEntry
    {
        public CacheContextEntry(string key, string value)
        {
            Key = key;
            Value = value;
        }

        public string Key { get; }
        public string Value { get; }
    }

Also I would prefer to not add an extra line between each field if there is no comment and defined in one line.

More globally, as already said somewhere, yes I don't like too compact code but now I also don't like code that is too evenly spaced, for example yes after each code block we add a new line but not necessarily before, for example to define a related local variable, this way the code is not too compact and it's easier to differentiate different parts.

Yes, only personal preferences, but as long as the analyzers we are using are not complaining, I would tend to respect what other people have already done.


---

Or, but here just as an example ;)

        public string Key { get; init; }
        public string Value { get; init; }

If we want to use something like this

    var entry1 = new CacheContextEntry("...", "...");
    var entry2 = entry1 with { Key = "..." };

Reply:
```
  public string Key { get; init; }
  public string Value { get; init; }
```

Is better as well